e2e-tests/IIoTPlatform-E2E-Tests/Standalone/A_PublishSingleNodeStandaloneDirectMethodTestTheory.cs

```diff
@@ -107,6 +111,23 @@ IIoTMultipleNodesTestContext context
 
             // Wait some time to generate events to process.
             await Task.Delay(TestConstants.DefaultTimeoutInMilliseconds, cts.Token);
+            //await Task.Delay(10000, cts.Token);
+
+            //Create request for GetConfiguredNodesOnEndpoint metod call
+            var NodesOnEndpoint = new PublishedNodesEntryModel();
```

Comment:
can we change the variable name to camel case?

Reply:
changed